src/benchmark/CMakeLists.txt

```diff
@@ -26,22 +26,6 @@ target_link_libraries(
     benchmark
 )
```

Comment:
Can't reply to https://github.com/hyrise/hyrise/pull/882#discussion_r202937370 so will do it here:

The views are templates in headers, they ARE in the same compilation unit.